app/src/main/java/com/fstyle/structure_android/MainActivity.java

```diff
@@ -8,6 +8,8 @@
  */
 public class MainActivity extends AppCompatActivity {
 
+    private int A = 1;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName